tests/block_test.go

```diff
@@ -28,6 +29,8 @@ func TestBlockchain(t *testing.T) {
 	bt.skipLoad(`^GeneralStateTests/`)
 	// Skip random failures due to selfish mining test
 	bt.skipLoad(`.*bcForgedTest/bcForkUncle\.json`)
+	// Skip these tests because the OVM gas limit will be different
```

Comment:
I think ideally the OVM gas limit will match Geth's gas limit because we will still need to make sure that the gas used in a single transaction is less than the block gas limit of Ethereum. The only thing that will be different is that we will also track a bit more information, in particular the number of unique SSTORES/SLOADS as well as the number of unique contracts touched.

Reply:
Tbh we were just super insane about the gas limit before because we were being lazy about testing & gas limit is the easiest thing to mess up & not realize.